pkg/devspace/kubectl/util.go

```diff
@@ -65,15 +66,22 @@ func EnsureDefaultNamespace(config *latest.Config, client kubernetes.Interface,
 	if defaultNamespace != "default" {
 		_, err = client.CoreV1().Namespaces().Get(defaultNamespace, metav1.GetOptions{})
 		if err != nil {
-			log.Donef("Create namespace %s", defaultNamespace)
-
 			// Create release namespace
 			_, err = client.CoreV1().Namespaces().Create(&v1.Namespace{
 				ObjectMeta: metav1.ObjectMeta{
 					Name: defaultNamespace,
 				},
 			})
+
+			log.Donef("Created namespace %s", defaultNamespace)
 		}
+	} else {
```

Comment:
This check isn't working as intended, since in the message the command `devspace use namespace` will change the default namespace but the warning will be printed again because configutil.GetDefaultNamespace will return default again